pkg/apis/gitopszombies/v1/types.go

```diff
@@ -0,0 +1,42 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Config defines a config for gitops-zombies.
+type Config struct {
+	metav1.TypeMeta `json:",inline"`
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec ConfigSpec `json:"spec,omitempty"`
+}
+
+// ConfigSpec configures an ingress class.
+type ConfigSpec struct {
+	Exclusions []Exclusion `json:"exclusions,omitempty"`
+}
```

Comment:
metadata and spec is not required, nothing is reconciled based on specs.

```suggestion
	metav1.TypeMeta `json:",inline"`
	Exclusions []Exclusion `json:"exclusions,omitempty"
````